src/test/resources/com/commercehub/gradle/plugin/avro/interop.avdl

```diff
@@ -45,6 +45,11 @@ protocol InteropProtocol {
     Kind enumField;
     MD5 fixedField;
     Node recordField;
+    decimal(10,2) decimalField;
+    date dateField;
+    time_ms timeField;
+    timestamp_ms timeStampField;
+    local_timestamp_ms localTimeStampField;
```

Comment:
Are there any other types that we might want coverage for?
It looks to me like we already have most of what is supported [here](https://github.com/apache/avro/blob/7d1e63b219e6d0778bc57195152477adee97fcab/lang/java/compiler/src/main/javacc/org/apache/avro/compiler/idl/idl.jj#L215).  Maybe `uuid`?

Reply:
I  think, that there is not any.
And `uuid` is not compilable with Avro 1.10.1.